.circleci/config.yml

```diff
@@ -98,7 +98,7 @@ jobs:
             . venv/bin/activate && mkdir packages
             set -eo pipefail
             # build main dash
-            git clone --depth 1 https://github.com/plotly/dash.git dash-main
+            git clone -b browser-capabilities --depth 1 https://github.com/plotly/dash.git dash-main
```

Comment:
@Marc-Andre-Rivet Are we planning to keep this (cloning from a branch) going forward, or just for testing here?

Reply:
This is just to validate that the changes in https://github.com/plotly/dash/pull/1507 don't break any tests. At least, not in ways we can't explain / fix.